spec/factories/theorem_factory.rb

```diff
@@ -3,7 +3,7 @@
     name { generate :sentence_name }
     text { Faker::Lorem.sentence }
     proof_status { create :proof_status_open }
-    state { 'pending' }
+    state { 'not_started_yet' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.